students/11th/daseulsong/js/login.js

```diff
@@ -0,0 +1,20 @@
+'user strict';
+
+const loginBox = document.querySelector('.login__main__idPw');
+const id = document.querySelector('.login__idPw--id');
+const pw = document.querySelector('.login__idPw--pw');
+loginBtn = document.querySelector('.login__btn--blue');
+[id, pw].forEach((item) => {
+  console.log(item);
+  item.addEventListener('keyup', (event) => {
+    console.log(event);
+    if (id.value !== '' && pw.value !== '') {
+      // 뭔가 입력되었을 때
+      loginBtn.style.backgroundColor = 'darkblue';
+    } else if (id.value === '' && pw.value === '') {
+      // 입력창에 아무것도 없을 때
+      console.log('working');
+      loginBtn.style.backgroundColor = 'rgb(95, 171, 243)';
+    }
+  });
```

Comment:
```suggestion
    if (id.value && pw.value) {
      loginBtn.style.backgroundColor = 'rgb(0, 0, 139)';
    } else if (!id.value && !pw.value) {
      loginBtn.style.backgroundColor = 'rgb(95, 171, 243)';
    }
  });
```
1. 자바스크립트에서 숫자 0, 빈 문자열("") 등은 false로 인식합니다. if문의 조건식에는 boolean 값이 사용된다는 점을 고려했을 때 "id.value가 빈 문자열이 아닐 때"라는 조건은 "id.value가 true일 때"라는 조건과 동치입니다. 즉 위와 같이 고쳐쓸 수 있습니다.
2. 색상은 가능하면 명확하게 hex 코드로 작성해주세요. rgb로 표현해두신 부분이 있어 일단 이 방식에 일치시켜두었습니다.
3. 테스트 끝난 console.log는 지워주세요.
4. 삼항연산자를 사용해볼 수 있습니다. 링크를 읽고 어떻게 적용할 수 있을지 고민해보세요.
(링크 : https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)

Reply:
피드백 1번 내용을 반영하여 조건식을 수정하였고, rgb 형식의 색상을 모두 hex로 변경하였습니다.
또한 삼항연산자 형태로 코드를 바꿔보았습니다.